satellite/console/consoleweb/consoleapi/README.md

```diff
@@ -0,0 +1,94 @@
+
+Requires setting `Authorization` header for requests.
+
+## API design
+
+### Successful responses
+
+For requests (`PUT`, `POST`, `DELETE`), endpoints should return an empty response body on success. `GET`
+requests can return a non-empty body for the resource that we're interacting with. 
+
+### Error responses
+
+When an API endpoint returns a client error (status code 4XX) it returns a JSON error response which contains 2 fields:
+
+* `error`: The error message.
+* `detail` (may be empty): Some detail about the returned error.
+
+Example:
+
+```json
+{
+  "error": "usage for the current month exists",
+  "detail": ""
+}
+```
+## Project Management API Endpoints
+
+Example of request
+```bash
+curl  -i -L \
+    -H "Accept: application/json" \
+    -H 'Authorization: <key>' \
```

Comment:
For clarity one could include the method: `-H 'Authorization: <method> <token>'`

Like `Authorization: Basic QWxhZGRpbjpvcGVuIHNlc2FtZQ==` or `Authorization: Bearer s07snpnaqiyla0fz1vjg2sam01j21z`